arch/xtensa/src/esp32s3/esp32s3_lowputc.c

```diff
@@ -795,6 +795,12 @@ void esp32s3_lowputc_config_pins(const struct esp32s3_uart_s *priv)
 {
   /* Configure the pins */
 
+  /* Keep TX pin in high level to avoid "?" trash character
+   * This "?" is the Unicode replacement character (U+FFFD)
+   */
+
+  esp32s3_gpiowrite(priv->txpin, true);
```

Comment:
Should this be located in scope of the `if` statement below?

Reply:
No, the issue happens in both case, be it using gpio matrix or native iomux. Should be something in the serial controller